ee/apps/omnichannel-transcript/src/initTranslation.ts

```diff
@@ -0,0 +1,22 @@
+import i18nDict, { availableTranslationNamespaces, defaultTranslationNamespace, extractTranslationNamespaces } from '@rocket.chat/i18n';
+import i18next from 'i18next';
+import sprintf from 'i18next-sprintf-postprocessor';
+
+const i18n = i18next.use(sprintf);
+
+void i18n.init({
```

Comment:
this will be third place we call `i18n.init` .. I understand the frontend will have different needs, but I was wondering if at least he backend usage could be shared (maybe moving the init to `@rocket.chat/i18n` package).. wdyt?

Reply:
Good question. To me, the PDF worker shouldn't know about translations, it should just do what it should do: pdf things. That's subjective however, so, wdyt?

Reply:
(you might be replying to my global comment, but I'll reply here anyways 😂 )
the point is that it actually knows about it https://github.com/RocketChat/Rocket.Chat/blob/0ff7a54f2e65406e5aef2081622307c0bb324975/ee/packages/pdf-worker/src/strategies/ChatTranscript.ts#L45

Reply:
> (you might be replying to my global comment, but I'll reply here anyways 😂 ) the point is that it actually knows about it
> 
> https://github.com/RocketChat/Rocket.Chat/blob/0ff7a54f2e65406e5aef2081622307c0bb324975/ee/packages/pdf-worker/src/strategies/ChatTranscript.ts#L45

About this one: that getTranslations func is weird... is not translating anything, its just finding the key on the translations we provide.

We could maybe look at if the pdf worker should handle all the translations, but i'd say it's more of a responsibility of whoever calls it to translate stuff. Or we could also take a look at that "translation" logic inside the pdfworker, cause it's not translating lol, its just finding.

Reply:
replied at https://github.com/RocketChat/Rocket.Chat/pull/36017#issuecomment-2887691430